swap/swap_test.go

```diff
@@ -687,24 +687,24 @@ func TestContractIntegration(t *testing.T) {
 	opts.Value = big.NewInt(0)
 	opts.Context = ctx
 
-	tx, err := issuerSwap.swapContract.Instance.SubmitChequeBeneficiary(
+	// we overwrite the waitForTx function with one which the simulated backend
```

Comment:
That's a lot of boilerplate to add to every test which involves contract interaction. Is there no before/after each test support in go?

Reply:
I added a `setupContractTest` function for this